src/SectorView.cpp

```diff
@@ -24,9 +20,13 @@
 #include "galaxy/Galaxy.h"
 #include "galaxy/Sector.h"
 #include "galaxy/StarSystem.h"
+#include "lua/LuaObject.h"
 #include "lua/LuaRef.h"
 #include "lua/LuaTable.h"
 #include "matrix4x4.h"
+#include <assert.h>
+#include <cmath>
+#include <sstream>
```

Comment:
Since clang-format is already nitpicking this, I'd move these includes down by one line so they're contiguous with the `#include <unordered_set>` statement and separated from `#include "matrix4x4.h".